class-constructor-new/exercise-1.js

```diff
@@ -20,15 +21,16 @@ export class Vector extends Array {
    * @returns {boolean}
    */
   equals(ve) {
-    if (!ve || !(ve instanceof Array)) {
-      throw new Error('Not a valid Vector');
-    }
-
-    return this.length === ve.length;
+    return this.isHavingSameLength(ve) && this.every((c, i) => c === ve[i])
```

Comment:
@viktordavidovich equals function compares each value of both vectors before yielding result.